README.md

```diff
@@ -1 +1,107 @@
 # avro-resolution_canonical_form
+
+This gem defines a [Resolution Canonical Form](#resolution_canonical_form) for Avro schemas.
+This is similar to the [Parsing Canonical Form](http://avro.apache.org/docs/1.8.1/spec.html#Parsing+Canonical+Form+for+Schemas)
+from the Apache Avro spec, but extends is to also include attributes that are
+relevant to schema resolution and compatibility.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'avro-resolution_canonical_form'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install avro-resolution_canonical_form
+
+## Resolution Canonical Form
+
+The Resolution Canonical Form extends the [Parsing Canonical Form](http://avro.apache.org/docs/1.8.1/spec.html#Parsing+Canonical+Form+for+Schemas)
+to include `default` and `aliases` attributes:
+
+* [ORDER] Order the appearance of fields in JSON objects as follows:
```

Comment:
The intent here was to extend the way that Parsing Canonical Form is described in the official spec. This adds `default` and `aliases` to the end of the list of fields, and specifies that `default` should appear before `aliases`.